### DISCUSSION TITLE
About Candidate Team Notification and Certificate Request

### DISCUSSION THREAD

**Discussion by zhoujingbo** [PaddlePaddle/PaddleSpatial#292] [BaiduKDDCup2022]
Dear Baidu KDD Cup 2022 Participants;

Thanks a lot for all your efforts to participate in the KDD Cup 2022! Please note the following information.

1) We have sent out the notifications to the award candidate teams (top-15 teams in Phase 3) for regular track and paddle track. If you do not receive the notification email, please send your team name to us ([kddcup2022@baidu.com](mailto:kddcup2022@baidu.com)), and we will check it.

2) If you want to obtain a certificate, please send a request to us before July 21. The organizer committee will provide a certificate of successful participation to your team if you can submit a report and the code no matter your rank.

3) We have **extended the paper submission deadline to 25th July**, and the award announcement time to 26th July.

Thanks a lot.

**Comment by shaido987** [comment]
Just to be clear: for participants that have already received an email notification there is no need to request a certificate, correct?

**Reply by zhoujingbo** [reply]
Yes. There is no need.

**Reply by shaido987** [reply]
Great. Thanks for the clarification.

**Comment by hongyingyue** [comment]
Thanks for the update.
So now the deadline of paper submission is 23:59 25th July (AOE time) ?

**Reply by zhoujingbo** [reply]
Yes. You are right.

**Comment by hansu1017** [comment]
We find that the submit results have been cleaned up, can we see the submit history again in order to enrich the experiment result in the report ? By the way, the code submission deadline is also 25th July, correct?

**Reply by zhoujingbo** [reply]
Yes, the code submission deadline is also 25th July. Unfortunately, the submit history is hidden. In the report, you can also report you offline test result to enrich the experiment result. Thanks ~
